src/components/views/rooms/LinkPreviewWidget.js

```diff
@@ -73,36 +73,13 @@ module.exports = React.createClass({
         this.unmounted = true;
     },
 
-    onImageClick: function(ev) {
-        var p = this.state.preview;
-        if (ev.button != 0 || ev.metaKey) return;
-        ev.preventDefault();
-        var ImageView = sdk.getComponent("elements.ImageView");
-
-        var src = p["og:image"];
-        if (src && src.startsWith("mxc://")) {
-            src = MatrixClientPeg.get().mxcUrlToHttp(src);
-        }
-
-        var params = {
-            src: src,
-            width: p["og:image:width"],
-            height: p["og:image:height"],
-            name: p["og:title"] || p["og:description"] || this.props.link,
-            fileSize: p["matrix:image:size"],
-            link: this.props.link,
-        };
-
-        Modal.createDialog(ImageView, params, "mx_Dialog_lightbox");
-    },
-
```

Comment:
removing the lightbox is a bit of a shame for previews of image urls... what was the rationale for removing it? i'd expect behaviour parity with viewing an image attachment